### DISCUSSION TITLE
User Feedback Emails Form - Sender email

### DISCUSSION THREAD

**Discussion by julien-paris** [code4recovery/12-step-meeting-list#1040] [General]
Hello all, 
- Regarding the meeting feedback form
- When a user fills it out the form and sends it it sends us an email
- previously, when we answered that email, the recipient was the email of the person who had filled the form. 

Since last update, this is not the case anymore, the recipient is wordpress@alcoholics-anonymous.eu. 

Upon checking, the headers do not contain the "reply-to: intial.senders@email.com"

This also is a problem due to the fact we have an autoresponder which now only goes to wordpress@alcoholics-anonymous.eu

Please advise, 

Thanks, 

Julien

**Comment by joshreisner** [comment]
Hi @julien-paris - we haven't made any changes to this area of the plugin in a long time, so I don't think it's related to a plugin update. We are still attaching the `Reply-To:` header to the emails from the plugin, but there are a lot of stages that come afterward that we don't control.

Are you using a plugin to send emails from your site? I just sent a test email.

**Reply by joshreisner** [reply]
I just did a test on our site, code4recovery.org, and the reply-to is being set properly. 

<img width="174" alt="Screenshot 2023-01-05 at 10 08 00 AM" src="https://user-images.githubusercontent.com/1551689/210850095-51a4baa5-b410-40c0-8252-610424187bbf.png">

I think it's safe to say the problem must be something happening after our plugin hands off to WordPress on your site. We invoke the `wp_mail` function, after that we don't control what happens - could be the reply-to is being stripped?

I wonder if trying this plugin might help you debug what's happening on the site: https://wordpress.org/plugins/check-email/

**Comment by julien-paris** [comment]
Hi Josh, thanks for the reply!
I do use Post smtp to log emails. 
I do not use it for its custom SMTP settings, we use the native Wordpress settings. 
I do have this message, which has been here for ages: 
"Postman (Pst SMTP) is not configured and is mimicking out-of-the-box WordPress email delivery."

I deactivated it and its now, oddly, picking up Mailpoet settings, and not sending to the addresses set in the settings. 

I will continue to fiddle around with it. 

Thanks, 

Julien

**Reply by joshreisner** [reply]
ok yeah please let us know if you find a good solution that works! 

i'm not sure how our demo site is sending email, pretty sure our host isn't helping at all. 

but if you want to try filling out the contact form on one of the meetings at code4recovery.org/meetings i'd be happy to forward it back to you so you can see that it is working there at least.

**Reply by julien-paris** [reply]
Thanks Josh, yes I saw the screenshot, very clear. It's also how it used to work for us. 
It just seems to be suddenly conflicting with other plugins which use the native mail service. 
Currently, he sender is the email associated with the Newsletter... it also only sends to the first recipient from the senttings.
Will keep you posted

**Comment by julien-paris** [comment]
Ok it is fixed - with new settings, not sure why. 
What did **not** work: disabling Mailpoet and/or Post SMTP  

What **did** work: 
In Post SMTP I switched to SMTP sending instead of the default WP settings. And mailer type is now PHP Mailer. 
This was random trial and and error, and not quite sure why the settings needs to be changed. For now all works. 
Thanks for the follow-up Josh, 
Take care
![image](https://user-images.githubusercontent.com/73740354/210903501-2ac50149-bb0b-4ef4-9e23-ac9c48711d8d.png)
